cmd/viam-agent/main.go

```diff
@@ -176,20 +167,46 @@ func main() {
 		}
 	}
 
-	// Start viam server as soon as possible. Then, start other subsystems and check for updates
-	if err := manager.StartSubsystem(ctx, viamserver.SubsysName); err != nil {
-		if errors.Is(err, agent.ErrSubsystemDisabled) {
-			globalLogger.Warn("viam-server subsystem disabled, please manually update /etc/viam.json and connect to internet")
-		} else {
-			globalLogger.Errorf("could not start viam-server subsystem: %s", err)
+	// if faststart is set, skip updates and start viam-server immediately, the proceed as normal
+	if opts.Fast {
```

Comment:
we have to let Pete know that tenibot will need to enable this

Reply:
Yeah, I definitely will let him know before this goes live, and will ask him to test and validate it still works for his use case before then too. I don't think they're shipping quite as urgently soon as canyon runner though, so hopefully it'll be a minor thing add. (The updates in the readme show exactly what to do to enable this.)

Reply:
Had a thought... this should be a config option in the cloud (as an attribute), not an ENV var. More work to implement it, but makes more sense that way, and allows control of this at fleet-level with upcoming fragments.